workers/unity/Assets/Playground/Scripts/Cubes/ScoreCollision.cs

```diff
@@ -0,0 +1,59 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Entities;
+using UnityEngine;
+
+[RemoveAtEndOfTick]
+public struct CollisionComponent : IComponentData
+{
+    public SpatialOSLaunchable own;
```

Comment:
If this is for an event entity should this be own and other? (Contrary to my other comment from when I thought this was being read on the entity themselves).
If this is being read on the spatial entity then this is fine.